migration-guides/v8.0.0.md

```diff
@@ -0,0 +1,29 @@
+# v8.0.0 Migration Guide
+
+This version contains the following breaking changes:
+
+1. Upgraded Node version to 18.12.1
+2. Removed `node-sass` in favor of `sass`
+3. Upgraded webpack and related loaders
+
+Further explanation of each item is detailed below.
+
+---
+
+## 1. Upgraded Node version to 18.12.1
+
+Typically, the Node version of an application can be updated without any breaking change. However, Node 18 requires an upgraded version of `glibc`. This has impact on both Travis and the existing `node-sass` executable.
+
+For Travis, the default distribution used for builds and test execution does not have the version of `glibc` required by Node 18. To resolve this issue, a specific distribution is specified in the `.travis.yml` configuration file that has the correct `glibc` version.
+
+`node-sass` has been deprecated and the latest version does not support the later `glibc` version (an exception is generated when one tries to execute `node-sass` with the later Node version). To resolve this, `node-sass` was removed and `sass` (Dart Sass, the currently active Sass implementation) was added. This change required an additional update to the `build:styles` script to use `sass` for style builds.
+
+Finally, this version of `lp-components` will not support older Node versions. You must use Node version ^18.12.1.
+
+## 2. Removed `node-sass` in favor of `sass`
+
+As mentioned above, `node-sass` was removed in favor of `sass`. Additionally, the latest version of `style-loader` was included in this version of `lp-components`.
```

Comment:
Is the `style-loader` note relevant in this section? Seems like your mention of it in #3 is sufficient enough IMO

Reply:
That's fine. Removed.